api/src/services/organizations/organizations.ts

```diff
@@ -74,6 +76,56 @@ export const getOrCreateOrganization = async (orgName, reportingPeriodName) => {
   }
 }
 
+export const downloadTreasuryFile: MutationResolvers['downloadTreasuryFile'] =
+  async ({ input }) => {
+    const { fileType } = input
+    const { organizationId } = context.currentUser.agency
+
+    // get the organization
+    const organization = await db.organization.findUnique({
```

Comment:
Do we want to throw an error here if organization is not found similar to the below?

Reply:
good call - done!